src/gateways/case-notes.test.js

```diff
@@ -0,0 +1,65 @@
+import moxios from 'moxios';
+import { CaseNotesGateway } from './case-notes.js';
+
+describe('Case notes gateway', () => {
+    const csGateway = new CaseNotesGateway();
+
+    beforeEach(() => {
+        moxios.install();
+    });
+
+    afterEach(() => {
+        moxios.uninstall();
+    });
+
+    it('200 Success "/residents/:residendId/caseNotes"', async () => {
+        // arrange
+        const randomId = Math.floor(Math.random() * 20);
+        const mockAxiosResponse = [
+            {
+                Id: 1,
+                CaseNote: 'repudiandae eos est',
+                HelpRequestId: 16,
+                ResidentId: randomId,
+                CreatedAt: '2020-12-25T16:15:41.325Z'
+            },
+            {
+                Id: 2,
+                CaseNote: 'vero qui voluptas',
+                HelpRequestId: 16,
+                ResidentId: randomId,
+                CreatedAt: '2021-01-20T05:06:34.567Z'
+            }
+        ];
+
+        const expectedResponse = [
```

Comment:
Could this just be `expectedResponse = mockAxiosResponse` 🤔

Reply:
Nevermind, just saw the casing

Reply:
It looks the same but The `mockAxiosResponse` comes back as Pascal Case and that gets changed to camelCase in the `ToResident` mapper so would need to reflect this in the test